src/lib/migrations/migrations.ts

```diff
@@ -71,8 +63,12 @@ export async function checkAndRunMigrations() {
 				continue;
 			}
 
-			// otherwise all is good and we cna run the migration
-			console.log(`[MIGRATIONS] "${migration.name}" not applied yet. Applying...`);
+			if (migration.runEveryTime) {
+				console.log(`[MIGRATIONS] "${migration.name}" should run every time. Applying...`);
+			} else {
+				// otherwise all is good and we cna run the migration
+				console.log(`[MIGRATIONS] "${migration.name}" not applied yet. Applying...`);
+			}
```

Comment:
```suggestion
            // otherwise all is good and we can run the migration
            console.log(`[MIGRATIONS] "${migration.name}" ${migration.runEveryTime ? "should run every time" : "not applied yet"}. Applying...`);
```